projects/mercury/src/collections/OrganisationBreadcrumbsContextProvider.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import {Folder} from "@material-ui/icons";
+import BreadcrumbsContext from "../common/contexts/BreadcrumbsContext";
+
+export default ({children}) => (
+    <BreadcrumbsContext.Provider value={{segments: [
+        {
+            label: 'Departments',
```

Comment:
Can you already make this configurable (depending on the model), instead of hardcoded entity?